### DISCUSSION TITLE
what is serverless.template.yml

### DISCUSSION THREAD

**Discussion by popmoore** [serverless/serverless#11394] [Q&A] [answered]
I found two config files: `serverless.template.yml` and `serverless.yml`. Who can tell me what's `serverless.template.yml`. What's the difference between `serverless.template.yml` and `serverless.yml` ?

**Comment by nick-zh** [comment]
I think this can be removed, from what i can see this is part of the [interactive setup](https://github.com/serverless/serverless/blob/3b3ccb0ea39a05c8fbb9353e7cef6e5626eecef1/lib/cli/interactive-setup/service.js#L258) and it tries to delete it at the end, but maybe something went wrong

**Reply by popmoore** [reply]
It seems that no official contributors take a look at this simple question and give me an answer. Thank you all the same.

**Comment by pgrzesik** [comment, marked as answer]
Hey @popmoore - `serverless.template.yml` were used in order to publish templates to internal Serverless registry. They don't have much use at the moment and you can disregard them. Some of the examples might still have it, but the file should be removed when setting up via interactive setup as @nick-zh explained.

**Reply by popmoore** [reply]
Thanks @pgrzesik
